samples/dnn/classification.cpp

```diff
@@ -113,101 +107,112 @@ int main(int argc, char** argv)
 
     //! [Open a video file or an image file or a camera stream]
     VideoCapture cap;
-    if (parser.has("input"))
-        cap.open(parser.get<String>("input"));
-    else
-        cap.open(0);
+    vector<string> imageFiles;
+    size_t currentImageIndex = 0;
+
+    if (parser.has("input")) {
+        string input = parser.get<String>("input");
+
+        if (input.find('.')==string::npos) {
```

Comment:
It may be network stream. I propose always use VideoCapture. It supports sequence of images too.

Reply:
@asmorkalov 
VideoCapture does support sequences of images, but it expects them to be in a numerically ordered format (e.g., image1.jpg, image2.jpg).  It cannot handle images with unordered prefixes (e.g., ab.jpg, bc.jpg, xyz.jpg). For cases where images are not named in a numerical sequence, this was implemented to read and process these images from a directory.

Reply:
We have imagelist approach is some samples. I propose to use it here too. See https://github.com/opencv/opencv/blob/d1b643a05a70a21355f4a9d3761de58f4d690b7b/samples/cpp/calibration.cpp#L301 for example. File format is described in the sample header.

Reply:
@asmorkalov Added string list. Can be used with --imglist parser option.